gradio/themes/utils.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+
+class Theme:
+    def _color(self, color: str, number: int = 500):
+        return f"var(--color-{color}-{number})"
+
+    def _use(self, property):
+        assert property in self.__dict__ and not property.endswith("_dark")
+        return f"var(--{property.replace('_', '-')})"
+
+    def _get_theme_css(self):
+        css = ":host, :root {\n"
```

Comment:
We can get rid of `:host` now we aren't use the shadow DOM. `:root` is sufficient.